src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3216,51 +3216,77 @@ static char* allocate_pages_individually(size_t bytes, char* addr, DWORD flags,
   return p_buf;
 }
 
-static size_t large_page_init_decide_size() {
+size_t os::large_page_init_decide_size() {
   // print a warning if any large page related flag is specified on command line
   bool warn_on_failure = !FLAG_IS_DEFAULT(UseLargePages) ||
-                         !FLAG_IS_DEFAULT(LargePageSizeInBytes);
+      !FLAG_IS_DEFAULT(LargePageSizeInBytes);
```

Comment:
There are still some unnecessary whitespace changes in this file

Reply:
Some context here: avoiding unnecessary whitespace changes reduces the size of the diff to be reviewed, makes git blame more helpful by not showing the last modification to the line as a whitespace change (which typically won't be the change you want to see when you run git blame), and reduces probability of merge conflicts when backporting since there are fewer lines changed.